.travis.yml

```diff
@@ -65,7 +65,7 @@ script:
   - |
     if [[ $TRAVIS_EVENT_TYPE != "cron" ]]; then
       if [[ $TOXENV == "amo-locales-and-signing" ]]; then
-         ./scripts/start-autograph.sh
+         docker run --name autograph -d -p 5500:5500 -v $(pwd)/scripts/:/scripts/ mozilla/autograph:2.0.5 /go/bin/autograph -c /scripts/autograph_travis_test_config.yaml
```

Comment:
This should be hitting `autograph:latest`, most certainly not `2.0.5` when `2.2.0` is current.